app/src/main/java/com/example/pocketcloset/Fragments/WardrobeFragment.java

```diff
@@ -1,65 +1,224 @@
+import com.example.pocketcloset.models.User;
+import com.google.android.material.button.MaterialButtonToggleGroup;
+import com.parse.FindCallback;
+import com.parse.ParseException;
+import com.parse.ParseQuery;
+import com.parse.ParseUser;
+
+import java.util.ArrayList;
+import java.util.List;
+
 
-/**
- * A simple {@link Fragment} subclass.
- * Use the {@link WardrobeFragment#newInstance} factory method to
- * create an instance of this fragment.
- */
 public class WardrobeFragment extends Fragment {
 
-    // TODO: Rename parameter arguments, choose names that match
-    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
-    private static final String ARG_PARAM1 = "param1";
-    private static final String ARG_PARAM2 = "param2";
+    protected ClothingAdapter adapter;
+    protected List<Clothing> allClothing;
 
-    // TODO: Rename and change types of parameters
-    private String mParam1;
-    private String mParam2;
+    private SwipeRefreshLayout swipeContainer;
+    private RecyclerView rvWardrobe;
+    public ParseUser user =  User.getCurrentUser();
+    public MaterialButtonToggleGroup toggleButton;
 
-    public WardrobeFragment() {
+    public WardrobeFragment(ParseUser userToFilterBy) {
         // Required empty public constructor
+        this.user =  userToFilterBy;
     }
 
-    /**
-     * Use this factory method to create a new instance of
-     * this fragment using the provided parameters.
-     *
-     * @param param1 Parameter 1.
-     * @param param2 Parameter 2.
-     * @return A new instance of fragment CameraFragment.
-     */
-    // TODO: Rename and change types and number of parameters
-    public static WardrobeFragment newInstance(String param1, String param2) {
-        WardrobeFragment fragment = new WardrobeFragment();
-        Bundle args = new Bundle();
-        args.putString(ARG_PARAM1, param1);
-        args.putString(ARG_PARAM2, param2);
-        fragment.setArguments(args);
-        return fragment;
-    }
 
-    @Override
-    public void onCreate(Bundle savedInstanceState) {
-        super.onCreate(savedInstanceState);
-        if (getArguments() != null) {
-            mParam1 = getArguments().getString(ARG_PARAM1);
-            mParam2 = getArguments().getString(ARG_PARAM2);
-        }
-    }
 
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container,
                              Bundle savedInstanceState) {
         // Inflate the layout for this fragment
-        return inflater.inflate(R.layout.fragment_camera, container, false);
+        return inflater.inflate(R.layout.fragment_wardrobe, container, false);
+    }
+
+    @Override
+    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+        rvWardrobe = view.findViewById(R.id.rvClothes);
+        toggleButton = view.findViewById(R.id.toggleButton);
+
+        allClothing = new ArrayList<>();
```

Comment:
Since you're always setting this to an empty array when the fragment is created, you can do it where the list is declared instead of here.